### PR TITLE
docs(config): add config usage suggestion

### PR DESCRIPTION
**Pull request by superhx**

### CODE REVIEW

File: core/src/main/scala/kafka/server/KafkaConfig.scala [hunk 1]
```diff
@@ -496,40 +490,52 @@ object KafkaConfig {
...
+    "You could also PoC launch it in memory mode with endpoint <code>memory:://</code> or redis mode with <code>redis://.</code>"
+  val ElasticStreamNamespaceDoc = "The kafka cluster in which elastic stream namespace which should conflict with other kafka cluster sharing the same elastic stream."
+  val S3EndpointDoc = "The object storage endpoint, ex. <code>https://s3.us-east-1.amazonaws.com</code>."
+  val S3RegionDoc = "The object storage region, ex. <code>us-east-1</code>."
+  val S3PathStyleDoc = "The object storage access path style. When using MinIO, it should be set to true."
+  val S3BucketDoc = "The object storage bucket."
+  val S3WALPathDoc = "The local WAL path for AutoMQ can be set to a block device path such as /dev/xxx or a filesystem file path." +
+    "It is recommended to use a block device for better write performance."
+  val S3WALCapacityDoc = "The size of the local WAL for AutoMQ. This determines the maximum amount of data that can be written to the buffer before data is uploaded to object storage." +
+    "A larger capacity can tolerate more write jitter in object storage. The value should be larger than s3.wal.cache.size"
```
Comment (Chillax-0v0) [inline core/src/main/scala/kafka/server/KafkaConfig.scala:505]: Now, there is no strict size relationship between `s3.wal.capacity` and `s3.wal.cache.size`.

**Review by Chillax-0v0 — DISMISSED** [review]

File: core/src/main/java/kafka/autobalancer/config/AutoBalancerControllerConfig.java [hunk 2]
```diff
@@ -68,21 +68,26 @@ public class AutoBalancerControllerConfig extends AbstractConfig {
...
+        "The actual time to check for the next data rebalancing is also related to the scale of the partition migration that occurred this time. The more partitions that are migrated, the longer the interval for the next data rebalancing check." +
+        "Shortening the minimum check interval can improve the sensitivity of data rebalancing, but in actual use, it should be configured in conjunction with the Broker metrics reporting interval to avoid the Controller being unable to perceive the result of the most recent scheduling, thereby causing duplicate scheduling.";
+    public static final String AUTO_BALANCER_CONTROLLER_NETWORK_IN_USAGE_DISTRIBUTION_DETECT_THRESHOLD_DOC = "Network in monitoring threshold. If it is below this threshold, the load of this Broker will not be actively optimized.";
+    public static final String AUTO_BALANCER_CONTROLLER_NETWORK_IN_DISTRIBUTION_DETECT_AVG_DEVIATION_DOC = "The NetworkInUsageDistributionGoal tries to adjust the network inbound bandwidth utilization rate of each broker to a traffic distribution range." +
```
Comment (SCNieh) [inline core/src/main/java/kafka/autobalancer/config/AutoBalancerControllerConfig.java:83]: The acceptable range of deviation for average network input bandwidth usage. Default is 0.2, which means the expected network traffic range will be [0.8 * loadAvg, 1.2 * loadAvg]

File: core/src/main/java/kafka/autobalancer/config/AutoBalancerControllerConfig.java [hunk 3]
```diff
@@ -68,21 +68,26 @@ public class AutoBalancerControllerConfig extends AbstractConfig {
...
+        "The default value is 0.2, which means the network inbound bandwidth utilization rate distribution range of each broker will be attempted to be adjusted to within the range of [average inbound bandwidth utilization rate * 0.8, average inbound bandwidth utilization rate * 1.2].";
+    public static final String AUTO_BALANCER_CONTROLLER_NETWORK_OUT_USAGE_DISTRIBUTION_DETECT_THRESHOLD_DOC = "Network out monitoring threshold. If it is below this threshold, the load of this Broker will not be actively optimized.";
```
Comment (SCNieh) [inline core/src/main/java/kafka/autobalancer/config/AutoBalancerControllerConfig.java:85]: 上同，改成 outbound

File: core/src/main/java/kafka/autobalancer/config/AutoBalancerControllerConfig.java [hunk 4]
```diff
@@ -68,21 +68,26 @@ public class AutoBalancerControllerConfig extends AbstractConfig {
...
     public static final String AUTO_BALANCER_CONTROLLER_NETWORK_OUT_DISTRIBUTION_DETECT_AVG_DEVIATION_DOC = "The acceptable range of deviation for average network output bandwidth usage";
```
Comment (SCNieh) [inline core/src/main/java/kafka/autobalancer/config/AutoBalancerControllerConfig.java:86]: 上同，改成 outbound

**Review by SCNieh — CHANGES_REQUESTED** [review]

**Review by SCNieh — APPROVED** [review]

**Pull request merged**